spec.html

```diff
@@ -40139,7 +40138,9 @@ <h1>AsyncFunctionStart ( _promiseCapability_, _asyncFunctionBody_ )</h1>
             1. Return.
           1. Push _asyncContext_ onto the execution context stack; _asyncContext_ is now the running execution context.
```

Comment:
This line should've been removed. Apologies for not verifying this better.

Reply:
I'll remove it directly. Also, that NOTE isn't quite what I had in mind, mind if I remove that as well? The invariant should hold of any [[Call]] internal method, not the Function one, and I was hoping to document that as such.

Reply:
the note is just explaining why there's a random push at the end there, but it's up to you.